native/src/main/java/org/ballerinax/asb/admin/Administrator.java

```diff
@@ -0,0 +1,999 @@
+import com.azure.messaging.servicebus.administration.models.RuleAction;
+import com.azure.messaging.servicebus.administration.models.RuleFilter;
+import com.azure.messaging.servicebus.administration.models.RuleProperties;
+import com.azure.messaging.servicebus.administration.models.SqlRuleAction;
+import com.azure.messaging.servicebus.administration.models.SqlRuleFilter;
+import com.azure.messaging.servicebus.administration.models.SubscriptionProperties;
+import com.azure.messaging.servicebus.administration.models.TopicProperties;
+import io.ballerina.runtime.api.PredefinedTypes;
+import io.ballerina.runtime.api.creators.TypeCreator;
+import io.ballerina.runtime.api.creators.ValueCreator;
+import io.ballerina.runtime.api.types.ArrayType;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.utils.TypeUtils;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BHandle;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import org.ballerinax.asb.util.ASBConstants;
+import org.ballerinax.asb.util.ASBErrorCreator;
+import org.ballerinax.asb.util.ASBUtils;
+import org.ballerinax.asb.util.ModuleUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static io.ballerina.runtime.api.creators.ValueCreator.createRecordValue;
+
+/**
+ * This facilitates the client operations of ASB Administrator client in Ballerina.
+ */
+public class Administrator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(Administrator.class);
+
+    /**
+     * Parameterized constructor for ASB Administrator (ServiceBusAdministrator).
+     *
+     * @param connectionString Azure service bus connection string
+     * @return clientEp Azure Service Bus Administrator instance.
+     */
+    public static Object initializeAdmin(String connectionString) {
+        try {
+            ServiceBusAdministrationClientBuilder administratorBuilder = new ServiceBusAdministrationClientBuilder()
+                    .connectionString(connectionString);
+            LOGGER.debug("ServiceBusAdministrator initialized");
+            return administratorBuilder.buildClient();
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+    }
+
+    /**
+     * Create a Topic in Azure Service Bus.
+     *
+     * @param administratorClient Azure Service Bus Administrator Client
+     * @param topicName           nameof the topic
```

Comment:
```suggestion
     * @param topicName           name of the topic
```

Reply:
Addressed in [0dd0f6c](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/0dd0f6ce7a0ad28cc7aaf39442d4edc19d47865d).